### ISSUE TITLE
Need to restart the console between "Browser Exploit" and "Making a NAND Backup" sections

### ISSUE THREAD

**Issue by budsterblue** [hacks-guide/Guide-WiiU#122]
Switching from CBHC to Tiramisu right now and got stuck for a minute trying to make a NAND backup (creating a new one because I misplaced my old one). After moving on from the "Browser Exploit" section, Tiramisu is loaded, which causes nanddumper to fail to load (just kicks you back to the Wii U menu). Rebooting before launching nanddumper fixes the issue, and 62faad9 seems to address the same issue later in the guide, so that same sentence (or something similar) should probably be added to the start of "Making a NAND Backup" as well.

TL;DR
Probably need the language from 62faad9 at the start of "Making a NAND Backup" as well.
